service/db/src/main/java/gov/va/vro/service/db/SaveToDbServiceImpl.java

```diff
@@ -87,14 +139,29 @@ private void insertAssessmentResult(
     claimRepository.save(claimEntity);
   }
 
+  @Override
+  public void setOffRampReason(Claim claimWithOffRamp) {
+    Optional<ClaimSubmissionEntity> claimSubmission =
+        claimSubmissionRepository.findFirstByReferenceIdAndIdTypeOrderByCreatedAtDesc(
+            String.valueOf(claimWithOffRamp.getCollectionId()), Claim.DEFAULT_ID_TYPE);
```

Comment:
This would be a bug for v2 endpoints.
```suggestion
            String.valueOf(claimWithOffRamp.getCollectionId()), claimWithOffRamp.getIdType());
```